core/src/cats/create_bareos_database.in

```diff
@@ -119,7 +119,7 @@ case ${db_type} in
       # This must be updated for future versions of PostgreSQL
       #
       case ${PSQLVERSION} in
-         9.*|10.*|11.*|12.*)
+         9.*|1[0123].*)
```

Comment:
Could you replace all lines 117-131 by: 
`ENCODING="ENCODING 'SQL_ASCII' LC_COLLATE 'C' LC_CTYPE 'C'"`

Reply:
done, exactly what I meant by "more drastically"

Reply:
Looking at that again, it actually could/should be 105-131 as PSQLVERSION is not needed anymore

Reply:
That's right. Good hint. Please remove this part, too.